src/base/dof_map_constraints.C

```diff
@@ -3149,9 +3081,8 @@ void DofMap::process_constraints (MeshBase & mesh)
   typedef std::set<dof_id_type> RCSet;
   RCSet unexpanded_set;
 
-  for (DofConstraints::iterator i = _dof_constraints.begin();
-       i != _dof_constraints.end(); ++i)
-    unexpanded_set.insert(i->first);
+  for (const auto & i : _dof_constraints)
+    unexpanded_set.insert(i.first);
 
   while (!unexpanded_set.empty())
     for (RCSet::iterator i = unexpanded_set.begin();
```

Comment:
Wow; finally I see a traditional for loop that couldn't be turned into a range for!